src/styles/main.scss

```diff
@@ -12,3 +12,12 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+@import "./utils/variables";
+
+@each $key, $val in $breakpoints {
+  @media (#{$val}) {
+    .catalog {
+      grid-template-columns: repeat($key, minmax(200px, 1fr));
```

Comment:
need to generalize it more , you can use variables instead of values
you can move it inside `catalog` block under `.catalog` class